newspack-sacha/sass/style.scss

```
@@ -72,12 +72,14 @@ Nicolas Gallagher and Jonathan Neal https://necolas.github.io/normalize.css/
 
 // Accent headers
 .accent-header,
+.widgettitle,
```

This should just apply when the `.widgettitle` class appears in the sidebar:

```suggestion
#secondary .widgettitle,
```